hardware/opas/TOPAS/TOPAS.py

```diff
@@ -176,27 +104,31 @@ def _load_curve(self, interaction):
             del curve_paths_copy['Poynting']
         crv_paths = [m.read() for m in curve_paths_copy.values()]
         all_curves = attune.TopasCurve.read_all(crv_paths)
+        for curve in all_curves.values():
+            for dependent in curve.dependent_names:
+                if dependent not in self.motor_names:
+                    try:
+                        curve.rename_dependent(dependent, self.motor_names[int(dependent)])
+                    except:
+                        pass
```

Comment:
Rename dependents in all the curves according to their index... The api calls are nice about names.

Try/except is because some curves reuse the same motors